# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/ContentParent.cpp
# Commit: 856a3d16a4ca
# Full Hash: 856a3d16a4ca856711ad9e6368cccd5aa1e22118
# Author: Nika Layzell <nika@thelayzells.com>
# Date: 2019-03-15 03:42:43
# Regressor Bug: 1532661
# File Overlap Count: 2
# Description:
#   Bug 1532661 - Part 5: Make the BrowsingContext opener edge a weak reference, r=farre
#   
#   Depends on D22193
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D23047
# ==============================================================================

diff -r 9ce1a2365aff -r 856a3d16a4ca dom/ipc/ContentParent.cpp
--- a/dom/ipc/ContentParent.cpp	Thu Mar 14 18:51:09 2019 +0000
+++ b/dom/ipc/ContentParent.cpp	Thu Mar 14 18:51:11 2019 +0000
@@ -5740,7 +5740,6 @@
     RefPtr<BrowsingContextGroup> group =
       BrowsingContextGroup::Select(aInit.mParentId, aInit.mOpenerId);
     child = BrowsingContext::CreateFromIPC(std::move(aInit), group, this);
-    child->InitFromIPC(aInit.mOpenerId);
   }
 
   child->Attach(/* aFromIPC */ true);
